client_end_user/shared/src/commonMain/kotlin/presentation/pickLanguage/PickLanguageScreen.kt

```diff
@@ -0,0 +1,78 @@
+package presentation.pickLanguage
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.lazy.grid.GridCells
+import androidx.compose.foundation.lazy.grid.LazyVerticalGrid
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.unit.dp
+import cafe.adriel.voyager.navigator.Navigator
+import com.beepbeep.designSystem.ui.theme.Theme
+import org.jetbrains.compose.resources.ExperimentalResourceApi
+import org.jetbrains.compose.resources.painterResource
+import presentation.base.BaseScreen
+import presentation.composable.LanguageCard
+import presentation.composable.HeadFirstCard
+import presentation.login.LoginScreen
+import resources.Resources
+
+
+object PickLanguageScreen :
+    BaseScreen<PickLanguageScreenModel, PickLanguageUIState, PickLanguageUIEffect, PickLanguageInteractionListener>() {
+    @Composable
+    override fun Content() {
+        initScreen(getScreenModel())
+    }
+
+    override fun onEffect(effect: PickLanguageUIEffect, navigator: Navigator) {
+        when (effect) {
+            is PickLanguageUIEffect.onGoToPreferredLanguage -> {}
+
```

Comment:
I think this is supposed to be "go to preferred food"

Reply:
yes i know but it not complete yet when i finish it i will navigat to it